<launch>

	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find crazypkg)/launch/consoleConfig.config"/>

		
	<include file="$(find crazypkg)/launch/CrazyRadio.launch" />

	<node pkg="crazypkg" name="ViconDataStreamSDK" output="screen" type="ViconDataStreamSDK">
		<rosparam command="load" file="$(find crazypkg)/launch/viconParams.yaml" />	
	</node>

	<node pkg="crazypkg" name="FlightControl" output="screen" type="FlightControl">
	
	</node>

	<node pkg="crazypkg" name="GUI" output="screen" type="GUI">
		<rosparam command="load" file="$(find crazypkg)/launch/ControlParams.yaml" />
	</node>

</launch>
